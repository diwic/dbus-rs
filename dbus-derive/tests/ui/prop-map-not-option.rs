use dbus_macros::DbusPropMap;

#[derive(DbusPropMap)]
pub struct PropsArg {
    pub arg1: String,
    pub arg2: i32,
}

fn main() {}
