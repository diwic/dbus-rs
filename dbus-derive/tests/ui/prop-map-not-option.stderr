error: dbus_derive - Will fail in case field is missing from dbus::PropMap
 --> tests/ui/prop-map-not-option.rs:5:15
  |
5 |     pub arg1: String,
  |               ^^^^^^
  |
  = help: Try this - Option<String>

error: dbus_derive - Will fail in case field is missing from dbus::PropMap
 --> tests/ui/prop-map-not-option.rs:6:15
  |
6 |     pub arg2: i32,
  |               ^^^
  |
  = help: Try this - Option<i32>
